coder_sniffer/Drupal/Test/Commenting/FunctionCommentUnitTest.php

```diff
@@ -79,6 +79,16 @@ public function getErrorList($testFile=null)
 
         case 'FunctionCommentUnitTest.2.inc':
             return [8 => 1];
+
+        case 'FunctionCommentUnitTest.3.inc':
+            if (PHP_VERSION_ID < 70200) {
+                return [
+                    9  => 1,
+                    14 => 1,
```

Comment:
I think it is not a good idea to have different test results per PHP version. Line 14 should never be flagged as error if the PHP type hint is object. We should just make "object" an acceptable type hint. People might run PHPCS on a lower PHP version and then get different results than on their CI or prod enviorinment. It is just an invitation for confusion.